apps/emqx_bridge_iotdb/src/emqx_bridge_iotdb_connector.erl

```diff
@@ -352,12 +373,12 @@ on_remove_channel(InstanceId, #{channels := Channels} = OldState0, ChannelId) ->
 on_get_channels(InstanceId) ->
     emqx_bridge_v2:get_channels_for_connector(InstanceId).
 
-on_get_channel_status(_InstanceId, ChannelId, #{channels := Channels}) ->
-    case maps:is_key(ChannelId, Channels) of
-        true ->
+on_get_channel_status(InstanceId, _ChannelId, State) ->
+    case on_get_status(InstanceId, State) of
+        connected ->
             connected;
         _ ->
-            {error, not_exists}
+            disconnected
```

Comment:
Please help adopt the `?status_disconnected`, `?status_connected` and other macros from `emqx_resource.hrl`, so usage is uniform and typo-proof.